docs/02.API-REFERENCE.md

```diff
@@ -3787,11 +3787,13 @@ is no longer needed.
 jerry_value_t
 jerry_exec_snapshot (const uint32_t *snapshot_p,
                      size_t snapshot_size,
+                     size_t func_index,
```

Comment:
FWIW, I would be interested in using it as a "library function", not as part of this CLI tool.